fixtures/apidocs_test_case.py

```diff
@@ -18,7 +18,7 @@ def create_schema(self):
             path = os.path.join(os.path.dirname(__file__), "../tests/apidocs/openapi-derefed.json")
             with open(path) as json_file:
                 data = json.load(json_file)
-                data["servers"][0]["url"] = settings.SENTRY_OPTIONS["system.url-prefix"]
+                data["servers"][0]["url"] = settings.SENTRY_OPTIONS["system.url-prefix"].rstrip('/')
```

Comment:
I don't think this is needed since this is a fixtures file and we default `system.url-prefix` to `http://testserver`

Reply:
good point, fixed and did commit --amend for my branch